### PR TITLE
Rename API to Uploading Docs

### PR DESCRIPTION
**Pull request by theaquarium**
Created a PR to talk about whether the documentation for uploading should be called an API. It's now just called `UPLOADING.md` and the README refers to this as "Uploading with API". Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,8 +82,8 @@ somecommand | upload
 somecommand | upload myfile.txt
 ```
 
-# API Documentation
-Want to make something that works with your uploader? You're in luck! We've got [API documentation](API.md).
+# Uploading with API
```
Comment (JoeGandy) [inline README.md:85]: Perhaps 'Uploading directly to your site'?

Comment (theaquarium) [reply README.md:85]: I feel like that's a bit wordy. How about "Direct Uploading"?
